plugin/PDFView.qml

```diff
@@ -32,9 +32,11 @@ SilicaFlickable {
     property alias currentPage: pdfCanvas.currentPage
 
     property bool scaled: pdfCanvas.width != width
+    property QtObject _feedbackEffect
 
     signal clicked()
     signal linkClicked(string linkTarget)
+    signal selectionClicked(string text)
```

Comment:
It's a remaining of later implementation to allow a context menu on selection tap (to ask for annotation creation on the selection for instance). It's removed from this PR.